2025-06-15 09:18:24,409 - INFO - ðŸš€ Using device: cpu
2025-06-15 09:18:24,412 - ERROR - Failed to load model from nucleusV4_model.pth: [Errno 2] No such file or directory: 'nucleusV4_model.pth'
2025-06-15 09:21:43,082 - INFO - ðŸš€ Using device: cpu
2025-06-15 09:21:43,085 - ERROR - Failed to load model from nucleusV4_model.pth: [Errno 2] No such file or directory: 'nucleusV4_model.pth'
2025-06-15 09:21:58,586 - INFO - ðŸš€ Using device: cpu
2025-06-15 09:21:58,588 - ERROR - Failed to load model from nucleusv4_model.pth: [Errno 2] No such file or directory: 'nucleusv4_model.pth'
2025-06-15 09:22:19,332 - INFO - ðŸš€ Using device: cpu
2025-06-15 09:22:19,336 - ERROR - Failed to load model from nucleusV4_model.pth: [Errno 2] No such file or directory: 'nucleusV4_model.pth'
2025-06-15 09:22:25,783 - INFO - ðŸš€ Using device: cpu
2025-06-15 09:22:25,790 - ERROR - Failed to load model from ../AI/nucleusV4_model.pth: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torchvision.models.efficientnet.EfficientNet was not an allowed global by default. Please use `torch.serialization.add_safe_globals([torchvision.models.efficientnet.EfficientNet])` or the `torch.serialization.safe_globals([torchvision.models.efficientnet.EfficientNet])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
2025-06-15 09:23:19,471 - INFO - ðŸš€ Using device: cpu
2025-06-15 09:23:19,482 - ERROR - Failed to load model from ../AI/nucleusV4_model.pth: Weights only load failed. This file can still be loaded, to do so you have two options, [1mdo those steps only if you trust the source of the checkpoint[0m. 
	(1) In PyTorch 2.6, we changed the default value of the `weights_only` argument in `torch.load` from `False` to `True`. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution. Do it only if you got the file from a trusted source.
	(2) Alternatively, to load with `weights_only=True` please check the recommended steps in the following error message.
	WeightsUnpickler error: Unsupported global: GLOBAL torchvision.models.efficientnet.EfficientNet was not an allowed global by default. Please use `torch.serialization.add_safe_globals([torchvision.models.efficientnet.EfficientNet])` or the `torch.serialization.safe_globals([torchvision.models.efficientnet.EfficientNet])` context manager to allowlist this global if you trust this class/function.

Check the documentation of torch.load to learn more about types accepted by default with weights_only https://pytorch.org/docs/stable/generated/torch.load.html.
